config-xml/addon/src/main/java/org/jboss/windup/config/parser/metadata/MetadataSourceFrameworkHandler.java

```diff
@@ -0,0 +1,31 @@
+package org.jboss.windup.config.parser.metadata;
+
+import org.apache.commons.lang.StringUtils;
+import org.jboss.windup.config.exception.ConfigurationException;
+import org.jboss.windup.config.metadata.TechnologyReference;
+import org.jboss.windup.config.parser.ElementHandler;
+import org.jboss.windup.config.parser.NamespaceElementHandler;
+import org.jboss.windup.config.parser.ParserContext;
+import org.w3c.dom.Element;
+
+@NamespaceElementHandler(elementName = "sourceFramework", namespace = "http://windup.jboss.org/v1/xml")
+public class MetadataSourceFrameworkHandler implements ElementHandler<TechnologyReference>
```

Comment:
Do we want to keep a distinction in XML about sourceFramework and sourcePlatform? @jsight  @OndraZizka ? in the UI selection it's just "source" and "target"